internal/plan/plan.go

```diff
@@ -89,6 +95,44 @@ func (s *Service) Equal(other *Service) bool {
 	return reflect.DeepEqual(s, other)
 }
 
+// ParseBackoff returns the parsed version of the backoff duration list, or
+// the default if it's not specified.
+func (s *Service) ParseBackoff() ([]time.Duration, error) {
+	if s.Backoff == nil {
+		return []time.Duration{
+			0,
+			10 * time.Second,
+			20 * time.Second,
+			40 * time.Second,
+			80 * time.Second,
+			160 * time.Second,
+			320 * time.Second,
```

Comment:
This is the default backoff list.

Reply:
factor = 2 :)

This seems too spaced out, btw, but we can tune later. People won't like waiting 10+ seconds until a process restarts after its first few exits.

Reply:
Yep, with the new values it's much faster: 0.5s, 1s, 2s, ... up to 30s.